src/wp-includes/media.php

```diff
@@ -5578,7 +5578,21 @@ function wp_get_loading_optimization_attributes( $tag_name, $attr, $context ) {
 	 * It is here to avoid duplicate logic in many places below, without having
 	 * to introduce a very specific private global function.
 	 */
-	$postprocess = static function( $loading_attributes, $with_fetchpriority = false ) use ( $tag_name, $attr, $context ) {
+	$postprocess = static function( $loading_attributes, $with_fetchpriority = false, $is_header = false ) use ( $tag_name, $attr, $context ) {
+		if ( $is_header ) {
+			/**
+			 * Filters the minimum square-pixels threshold for an image to be eligible as the high-priority image.
+			 *
+			 * @since 6.3.0
+			 *
+			 * @param int $threshold Minimum square-pixels threshold. Default 50000.
+			 */
+			$wp_min_priority_img_pixels = apply_filters( 'wp_min_priority_img_pixels', 50000 );
```

Comment:
No need to duplicate the filter documentation, in core duplicate calls to a filter typically use markup like the following:
```suggestion
			/** This filter is documented in wp-admin/includes/media.php */
			$wp_min_priority_img_pixels = apply_filters( 'wp_min_priority_img_pixels', 50000 );
```

Reply:
Thank you, I wasn't aware of this. I'll keep this in mind for next time. 
Just a bit curious - Should we have done the same for ['wp_img_tag_add_loading_attr'](https://github.com/WordPress/wordpress-develop/pull/4495/files#diff-4d28b6652f0dab208c19aa45e1cf09f3c967060d8cc77502bfcd473a3216b612R1978) filter. I do understand the older function was moved to deprecated there, so made sense to add the documentation again.

Reply:
@kt-12 Good point! Let's keep the documentation in the non-deprecated function only. Can you update the deprecated function to use that "This filter is documented in ..." text please? Feel free to do it in the same PR.

Reply:
Fixing this in #4756.